### DISCUSSION TITLE
Is it possible to use already existing snapshots?

### DISCUSSION THREAD

**Discussion by janm84** [psy0rz/zfs_autobackup#271] [Q&A]
Hello,

on one of my systems I use incus in combination with zfs to store container instances and data volumes.
In addition, incus itself regularly creates snapshots.

I am looking for a tool that uses these existing snapshots to back them up to an external drive or remotely. 

Is it possible to conifigure zfs_autobackup in that way, that it does not create snapshots itself, but uses the existing snapshots from incus? 

I'm looking forward to your feedback. 

BR, Jan

**Comment by psy0rz** [comment]
yes, use `--other-snapshots`: https://github.com/psy0rz/zfs_autobackup/wiki/Manual#step-34-transferring-snapshots
